Sources/StreamChat/Workers/EventNotificationCenter_Tests.swift

```diff
@@ -26,8 +26,9 @@ final class EventNotificationCenter_Tests: XCTestCase {
             .init()
         ]
         
-        // Create notication center with middlewares
-        let center = EventNotificationCenter(middlewares: middlewares, database: database)
+        // Create notification center with middlewares
+        let center = EventNotificationCenter(database: database)
+        middlewares.forEach(center.add)
```

Comment:
Maybe we can just introduce a method for adding multiple middlewares at once? ;-)